src/map/battle.c

```diff
@@ -4032,12 +4032,14 @@ static struct Damage battle_calc_misc_attack(struct block_list *src, struct bloc
 	case CR_ACIDDEMONSTRATION:
 #ifdef RENEWAL
 		{// [malufett]
+			// [violetharmony] ratio and def change
 			int64 matk=0, atk;
 			short tdef = status->get_total_def(target);
 			short tmdef =  status->get_total_mdef(target);
 			int targetVit = min(120, status_get_vit(target));
 			short totaldef = (tmdef + tdef - ((uint64)(tmdef + tdef) >> 32)) >> 1; // FIXME: What's the >> 32 supposed to do here? tmdef and tdef are both 16-bit...
-
+			totaldef = totaldef/2;
```

Comment:
are you really sure? 
`>> 1` is equivalent to `/2`, so basically formula divides by 4?